spinalcordtoolbox/scripts/sct_run_batch.py

```diff
@@ -174,6 +175,33 @@ def _find_nonsys32_bash_exe():
     return shutil.which('bash', path=nonsys32_paths)
 
 
+def _filter_directories(dir_list, include=None, include_list=None, exclude=None, exclude_list=None):
+    # Handle inclusion lists
+    if include is not None:
+        dir_list = [f for f in dir_list if re.search(include, f) is not None]
```

Comment:
The result of `re.search` is guaranteed to be truthy if it's not None, so you can drop the `is not None` if you want.